### DISCUSSION TITLE
Projects Beta

### DISCUSSION THREAD

**Discussion by xmljim** [community/community#17117] [Projects and Issues]
Overall, it's very good.  One thing I'd love to see is the ability to display the Issue number along with the issue title, so that when I create commits against that issue, I don't have to hover over the link to see the issue number so that I can put it in my commit description (e.g., GH-20)

**Comment by GMNGeoffrey** [comment]
Another use case would be a stable sorting order for issues. I would like issue number to be one of the columns. I have no idea what the number field is useful for and would much prefer to just have the issue number there

**Comment by tom-os** [comment]
This is a problem for us also, mentioned by multiple users.
